core-common/src/main/java/org/zalando/nakadi/domain/NakadiRecordResult.java

```diff
@@ -3,26 +3,38 @@
 public class NakadiRecordResult {
 
     public enum Status {
-        SUCCEEDED, FAILED, NOT_ATTEMPTED
+        SUCCEEDED, FAILED, ABORTED
+    }
+
+    public enum Step {
+        NONE,
+        VALIDATION,
+        ENRICHMENT,
+        PARTITIONING,
```

Comment:
The actual order of steps is the opposite.  Let's change it to avoid confusion:
```suggestion
        PARTITIONING,
        ENRICHMENT,
```